specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/HybridLinks.json

```diff
@@ -0,0 +1,460 @@
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "The specified hybrid link does not exist."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Initiate hybrid link drop.": {
+            "$ref": "./examples/HybridLinkDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "HybridLinkListResult": {
+      "description": "A list of hybrid links in instance.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HybridLink"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "HybridLinkProperties": {
+      "description": "The properties of a hybrid link.",
+      "type": "object",
+      "properties": {
+        "targetDatabase": {
+          "description": "The name of the target database",
+          "type": "string"
+        },
+        "sourceEndpoint": {
+          "description": "The source endpoint",
+          "type": "string"
+        },
+        "primaryAvailabilityGroupName": {
+          "description": "The primary availability group name",
+          "type": "string"
+        },
+        "secondaryAvailabilityGroupName": {
+          "description": "The secondary availability group name",
+          "type": "string"
+        },
+        "replicationMode": {
+          "description": "The replication mode of hybrid link. Parameter will be ignored during link creation.",
+          "enum": [
+            "Async",
+            "Sync"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ReplicationMode",
+            "modelAsString": true
+          }
+        },
+        "distributedAvailabilityGroupId": {
+          "format": "uuid",
+          "description": "The distributed availability group id",
+          "type": "string",
+          "readOnly": true
+        },
+        "sourceReplicaId": {
+          "format": "uuid",
+          "description": "The source replica id",
+          "type": "string",
+          "readOnly": true
+        },
+        "targetReplicaId": {
+          "format": "uuid",
+          "description": "The target replica id",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Resource": {
```

Comment:
>Resource [](start = 5, length = 8)

Please refer to common definitions "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/..."